stdlib/prelude.kl

```diff
@@ -11,7 +11,7 @@
 (export -- primitive types
         Syntax
         ->
-        Integer
+        Integer + - * / abs negate > >= < <= = /=
```

Comment:
`integer->string` is again omitted. probably a mistake?

Reply:
Yep :)